torchinfo/layer_info.py

```diff
@@ -30,6 +30,10 @@ def __init__(
         self.executed = False
         self.parent_info = parent_info
         self.var_name = var_name
+        self.leaf_layer = True
+        for name, param in self.module.named_parameters():
+            if "." in name:
```

Comment:
Can you explain why a period guarantees the layer is a leaf_layer? It seems better to check whether the layer has any children instead.

Reply:
As far as I know, period is in the name of child layers. But your suggestion seems more correct. I will update the judgement.